### ISSUE TITLE
Question about the license

### ISSUE THREAD

**Issue by Bill2462** [openai/CLIP#203]
This repository is licensed under MIT license but model weights are downloaded from external source. Does that mean that weights are also licensed under MIT?

**Comment by anirudth** [comment]
I agree that the models have bias and corner cases. But is there clarity if weights of CLIP can be used for commercial use albeit in a controlled setting?

**Comment by metalwhale** [comment]
Unfortunately the weights are not intended for commercial use.
From [model card](https://github.com/openai/CLIP/blob/main/model-card.md#out-of-scope-use-cases)
> Any deployed use case of the model - whether commercial or not - is currently out of scope.
